stable/redis/README.md

```diff
@@ -131,6 +131,7 @@ The following table lists the configurable parameters of the Redis chart and the
 | `slave.securityContext.fsGroup`            | Group ID for the container (redis slave pod)                                                                   | `master.securityContext.fsGroup`                     |
 | `slave.securityContext.runAsUser`          | User ID for the container (redis slave pod)                                                                    | `master.securityContext.runAsUser`                   |
 | `slave.resources`                          | Redis slave CPU/Memory resource requests/limits                                                                | `master.resources`                                   |
+| `slave.podAntiAffinity`                          | Enable pod podAntiAffinity for slaves                                                                | `false`                                   |
```

Comment:
I'd prefer to see an `affinity` option instead, which the user can set themselves. This is what most other Charts seem to be moving to and it gives a user some more freedom. You could put your example (which should be good enough for 90% of all cases out there) in the README instead.

Reply:
Like the option to have a soft or hard affinity?

Reply:
Indeed, or to use different host labels. You can check `stable/mongodb-replicaset` for an example on how to do it in the Chart itself.